### PR TITLE
Copy of LICENSE in repo root

### PR DESCRIPTION
**Pull request by cpswan**
**- Why?**
Make the LICENSE visible at the top level of the repo, which will also appear in GitHub sidebar like:
image

**- What I did**
Copy of LICENSE from one of the projects into repo root

**- How I did it**
`cp at_backupkey_flutter/LICENSE .`

**- Description for the changelog**
>Copy of LICENSE in repo root

### CODE REVIEW

**Comment by cpswan** [comment]
@murali-shris I'm still waiting on your approval for this

**Comment by cpswan** [comment]
Some of the pubspec.yaml files already had homepage and repository set thus:

```yaml
homepage: https://atsign.dev
repository: https://github.com/atsign-foundation/at_widgets
```

This PR now makes them all consistent, which fixes #53 (it wasn't only at_events_flutter, there were a bunch of others pointing to the master branch).

**Review by sarika01 — APPROVED** [review]

**Comment by cpswan** [comment]
@sarika01 these packages will need to be pushed to pub.dev for the docs changes to take effect. Anything that's going to be updated in the next week or so can be swept along with the pushes that will happen anyway. For anything else can you please do a version+1 push.

**Pull request merged**
